### PR TITLE
Section title indication in app

### PR DESCRIPTION
**Pull request by ipavlopoulos**
Fetch conversation from comment id
Show comment with section/page title and/or preceding comments.

### CODE REVIEW

File: view_conversations/conv-viewer-webapp/src/app/app.component.ts [hunk 1]
```diff
@@ -289,4 +278,36 @@ export class AppComponent implements OnInit {
...
+            .get(encodeURI(
+              '/api/comment-id/' + searchFor))
+            .subscribe(
+                (comments: wpconvlib.Comment[]) => {
+                  console.log('got comments!');
+                  this.browseResult = JSON.stringify(comments, null, 2);
+                  delete this.inFlightBrowseRequest;
+                  console.log(comments);
+                  this.answerComments = comments.filter(i => i.id === searchFor);
+                  /*this.answerComments[0].isCollapsed=true;*/
```
Comment (sorensenjs) [inline view_conversations/conv-viewer-webapp/src/app/app.component.ts]: please remove commented code.

File: view_conversations/conv-viewer-webapp/src/app/app.component.ts [hunk 2]
```diff
@@ -289,4 +278,36 @@ export class AppComponent implements OnInit {
 
   }
 
+  /**
+  * Fetch the so-far conversation given a comment ID
```
Comment (sorensenjs) [inline view_conversations/conv-viewer-webapp/src/app/app.component.ts]: add period

File: view_conversations/conv-viewer-webapp/src/app/conversation/conversation.component.html [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
-      <td class="whenandwho">
-        <div class="author" *ngIf="rootComment.user_text">{{rootComment.user_text}}</div>
-        <div class="anonAuthor" *ngIf="!rootComment.user_text">Anonymous</div>
-        <div class="timestamp">{{rootComment.timestamp}}</div>
-      </td>
-      <td class="content">{{rootComment.cleaned_content}}</td>
+      <div *ngIf="rootComment.type=='CREATION'; then thenBlock else elseBlock"></div>
+        <ng-template #thenBlock>
+          <!-- This is appeared next to Page Title -->
+          <!--td class="whenandwho">
```
Comment (iislucas) [inline view_conversations/conv-viewer-webapp/src/app/conversation/conversation.component.html]: lots of commented out stuff here, remove?

**Comment by iislucas** [comment]
In description can you include screenshot? (helps understand what it is doing, and also verify CSS more easily).

**Pull request merged**